SRC/element/adapter/TclAdapterCommand.cpp

```diff
@@ -198,9 +198,9 @@ int TclModelBuilder_addAdapter(ClientData clientData, Tcl_Interp *interp,  int a
     
     // now create the adapter and add it to the Domain
     if (mass == 0)
-        theElement = new Adapter(tag, nodes, dofs, kb, ipPort, doRayleigh);
+        theElement = new Adapter(tag, nodes, dofs, kb, ipPort, 0, 0, doRayleigh);
     else
-        theElement = new Adapter(tag, nodes, dofs, kb, ipPort, doRayleigh, mass);
+        theElement = new Adapter(tag, nodes, dofs, kb, ipPort, 0, 0, doRayleigh, mass);
     

```

Comment:
These calls to the constructor are neglecting the `int ssl` and `int udp` parameters. The prototype of the Adapter's constructor is:
```cpp
    Adapter(int tag, ID nodes, ID *dof, const Matrix &stif,
        int ipPort, int ssl = 0, int udp = 0,
        int addRayleigh = 0, const Matrix *mass = 0);
```